test/jasmine/assets/unpolyfill.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Un-polyfills - to be included in karma.conf.js to catch
+ * browser-dependent errors.
+ */
+
+'use strict';
+
+(function(arr) {
+    arr.forEach(function(item) {
```

Comment:
Taken from @alexcjohnson 's https://github.com/plotly/plotly.js/issues/3186#issuecomment-434418436, this gave 

![image](https://user-images.githubusercontent.com/6675409/47743360-47384100-dc55-11e8-9342-ef9c30d94f4e.png)

before the previous commit.

I chose to throw instead of logging to avoid situations like the one solved in https://github.com/plotly/plotly.js/pull/3168.

Reply:
Note: we could also add `unpolyfill.js` to our image tests (like we do for [`strict-d3`](https://github.com/plotly/plotly.js/blob/master/test/image/strict-d3.js)).

I chose not to as throwing errors in our jasmine test should be enough to catch these bugs and debugging the old nw.js image server is a pain.